app/src/main/java/com/celzero/bravedns/service/BraveVPNService.kt

```diff
@@ -1612,7 +1696,8 @@ class BraveVPNService :
         // restart vpn to add new routes if in ipv46 mode
         if (appConfig.getInternetProtocol().isIPv46() && isRoutesChanged) {
```

Comment:
must restart vpn when user-preferred route changes, too.

Reply:
Yes that true but do it work better